pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -148,6 +150,7 @@ rustPlatform.buildRustPackage rec {
     protobuf
     rustPlatform.bindgenHook
     cargo-about
+    makeWrapper
```

Comment:
This should be wrapped by `lib.optional stdenv.hostPlatform.isLinux` (although it builds fine on darwin either way, just to avoid an unnecessary dependency)

Reply:
Done. Adding nodejs to the build inputs directly is not necessary, right?